tiles.js

```diff
@@ -0,0 +1,130 @@
+(function(){
+
+  $(document).ready(function(){
+    var Tile = function (){
+        this.inner = '#48895e',
+        this.outer = '#34495e',
+        this.current = this.outer
+        this.height = 200,
+        this.width =  200,
+        this.element = function() {
+            var div = document.createElement("div");
+            $(div).css('background-color', this.current);
+            $(div).css('height', this.height).css('width', this.width).css('margin', 2);
+            return div;
+        };
+    };
+    
+
+    Tile.prototype = {
+      build: function() {
+            var div = document.createElement("div");
+            $(div).css('height', this.height).css('width', this.width).css('margin', 2);
+            $(div).css('background-color', this.current)
+            return div;
+        },
+      toggleColor: function() {
+        if (this.current === this.inner) {
+            this.current = this.outer;
+        } else {
+            this.current = this.inner;
+        }
+      },
+      changeColor: function() {
+        this.toggleColor();
+        return this.build();
+      },
+      delayChangeColor: function() {
+        this.toggleColor();
+        setTimeout('this.changeColor()', 2000);
```

Comment:
I hadn't ever seen this syntax before...FYI that it's deprecated and has some drawbacks.

http://stackoverflow.com/questions/6081560/is-there-ever-a-good-reason-to-pass-a-string-to-settimeout